package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "ember-cli-version-checker": "^1.2.0",
     "execa": "^0.6.0",
-    "fs-extra": "^2.0.0",
+    "fs-extra": "^3.0.0",
```

Comment:
This is the typical example where I don't think increasing the major version has any benefit. We know eslint-config-simplabs works with 2.x so there's no need to increase to 3.x. What we could do is change this so that it allows both 2.x and 3.x which would be

```json
"fs-extra": "2 - 3"
```

What do you think @Turbo87?

Increasing the major version only increases people's node_modules folders as they now need to install `fs-extra` 2.x *and* 3.x where only 2.x *or* 3.x would be sufficient.